server/service/src/sync/api/get_site_status.rs

```diff
@@ -11,6 +11,8 @@ pub(crate) struct SiteStatusV5 {
 
 #[derive(Debug, PartialEq, Serialize, Deserialize)]
 pub(crate) enum SiteStatusCodeV5 {
+    #[serde(rename = "sync_is_running")]
```

Comment:
I noticed this status wasn't actually being mapped correctly from: https://github.com/openmsupply/msupply/blob/master/Project/Sources/Methods/siteGetSyncStatus.4dm